src/styles/index.scss

```diff
@@ -706,3 +731,11 @@ div.el-loading-parent--relative {
 .el-drawer__header .drawer-title {
   color: var(--color-text-primary);
 }
+
+.margin-top-10 {
+  margin-top: 10px;
+}
+
+.margin-top-20 {
+  margin-top: 20px;
+}
```

Comment:
I'm sorry, but I can't see any discrepancies from your description since there's no actual code to review. Please share the code you'd like checked if it includes anything specific that needs looking at, such as inconsistencies between classes or styles used across components.